pages/purchased.js

```diff
@@ -1,3 +1,47 @@
+import ShoppingList from "@/components/ShoppingList";
+import useSWR from "swr";
+
+const fetcher = (url) => fetch(url).then((response) => response.json());
+
 export default function PurchasedPage() {
-  return <p>This will show purchased items</p>;
+  const { data, mutate } = useSWR("/api/items", fetcher);
```

Comment:
Now that you have the global fetcher in your `_app.js`, you can remove the fetcher as an argument:

```suggestion
  const { data, mutate } = useSWR("/api/items");
```